### ISSUE TITLE
Accessibility checks are different with the same story

### ISSUE THREAD

**Issue by Lucasbeneston** [storybookjs/storybook#17573]
**Describe the bug**
Accessibility checks are different for the same url with Chrome 98.0.4758 and Safari 15.0. 
It's work on Firefox 97.0.1

**To Reproduce**

Link to the storybook : https://61fd537ecf081f003a135235-nndgxdlrso.chromatic.com/?path=/story/button--inaccessible

Step 1 : I click on the url and I see the accessibility checks  of the first component (button-inaccessible) with 
`1 violations | 4 assists | 0 incomplete`

Step 2 : I click on another component (ex: button-accessible) and I come back to the first one (button-inaccessible). 

Step 3 :  I have different values for the first component and all the other components have the same accessibility checks values : 
`0 violations | 1 assist | 0 incomplete`

**System**
```
Environment Info:

  System:
    OS: macOS 11.6
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 16.13.2 - ~/.nvm/versions/node/v16.13.2/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v12.22.1/bin/yarn
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.2/bin/npm
  Browsers:
    Chrome: 98.0.4758.102
    Firefox: 97.0.1
    Safari: 15.0
  npmPackages:
    @storybook/addon-a11y: ^6.4.18 => 6.4.19 
    @storybook/addon-actions: ^6.4.18 => 6.4.18 
    @storybook/addon-essentials: ^6.4.18 => 6.4.18 
    @storybook/addon-links: ^6.4.18 => 6.4.18 
    @storybook/addon-react-native-web: ^0.0.18 => 0.0.18 
    @storybook/react: ^6.4.18 => 6.4.18 
```

**Additional context**
PR : https://github.com/pass-culture/pass-culture-app-native/pull/2694
